srcpkgs/gpodder/template

```diff
@@ -1,18 +1,19 @@
 # Template file for 'gpodder'
 pkgname=gpodder
-version=3.10.17
+version=3.10.21
 revision=1
 wrksrc="gpodder-${version}"
-hostmakedepends="python3 intltool"
+hostmakedepends="python3 intltool which"
```

Comment:
Isn't which needed only for check?

Reply:
Yep, will add it to checkdepends.

Reply:
My point was that then which can be removed from hostmakedepends or does the build need it?

Reply:
Only the test suite needs it, in order to find the python interpreter and possibly other stuff.

Reply:
I think you forgot to drop it from hostmakedepends then.

Reply:
You are right ofc. Fixed.